### DISCUSSION TITLE
webcam wcinit fails with release 10 but works with 9.5.0

### DISCUSSION THREAD

**Discussion by joba-1** [arendst/Tasmota#13563] [Q&A] [answered]
Hi, 

I just flashed the current precompiled tasmota32-webcam release 10 with esptool.py on a new esp32-cam and got errors on wcinit (105 and 0xffffffff iirc).
I downgraded ota to release 9.5.0 while keeping settings and the cam worked fine.

Known issue? Couldn’t find one.

Unfortunately I killed the esp while trying to focus the lense so I cannot follow up with logs or test if it just was a bad first flash :(

**Comment by sfromis** [comment, marked as answer]
Not a general issue, just saw my face via tasmota32-webcam.bin v10.0.0 on an ESP32-CAM board.
